_posts/2022-05-30-reached-50000-users.md

```diff
@@ -0,0 +1,13 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Usegalaxy.eu has reached 50,000 registered users'
+tags: [galaxy, support, society, report]
+supporters:
+- galaxy-europe
+---
+
+At the end of 2021, we celebrated the 40,000 registered users. Only a few months later, we celebrate another milestone: 50,000 users! The European Galaxy team would like to thank you all for using Galaxy!
```

Comment:
That one is a little bit uninspired, can we postpone that one or work on it a bit more.

Reply:
moved to new PR